### PR TITLE
Files for topic circuitpython

### PR DESCRIPTION
**Pull request by TheKitty**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

>  This adds context to the existing tag circuitpython

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by nokiae0106** [comment]
Foo

**Comment by Sohailsohail30182** [comment]
gh pr checkout 2968

File: topics/circuitpython/circuitpython.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (MikeMcQuaid) [inline topics/circuitpython/circuitpython.md:1]: @TheKitty Whoops, this duplicates the `index.md` below and can be deleted!

File: topics/circuitpython/index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+aliases: circuit-python, circuit python
```
Comment (MikeMcQuaid) [inline topics/circuitpython/index.md:2]:
```suggestion
aliases: circuit-python
```

These can't have spaces in them unfortunately!

**Review by MikeMcQuaid — COMMENTED** [review]
Thanks for the PR, great work so far @TheKitty! A few minor changes and we'll get this shipped!

**Review by seniordiari1 — COMMENTED** [review]
Nicest

**Review by seniordiari1 — COMMENTED** [review]
thekitty:main

**Review by Mankan777 — COMMENTED** [review]
2244.DU

**Review by Mankan777 — COMMENTED** [review]
2244.ME

**Review by Rugetz — COMMENTED** [review]
https://vt.tiktok.com/ZSdUH2HM4/

**Pull request closed without merging**